test/e2e/lifecycle/upgrade_test.go

```diff
@@ -18,6 +18,7 @@ var _ = Context("Cluster Network Addons Operator", func() {
 		Context(fmt.Sprintf("when operator in version %s is installed and supported spec configured", oldRelease.Version), func() {
 			BeforeEach(func() {
 				InstallRelease(oldRelease)
+				CheckCrdExplainable()
```

Comment:
Why do we care about this in before each?

Reply:
the idea was to check before and after the upgrade. 
I agree this would better suited in a specific `It`. tell me what you think
Done